package.json

```diff
@@ -39,6 +39,8 @@
   "dependencies": {
     "canvas": "^1.4.0",
     "map-limit": "0.0.1",
+    "maxrects-packer": "^1.0.2",
+    "js2xmlparser": "^3.0.0",
```

Comment:
@arilotter Sorry about that, it was leaked from another branch (Export xml format)